proposals/0269-New-vehicle-data-ClimateData.md

```diff
@@ -25,7 +25,7 @@ We need to add `ClimateData` for `GetVehicleData`, `SubscribeVehicleData`, `Unsu
 <element name="VEHICLEDATA_CLIMATEDATA" since="X.x"/>
 <element name="VEHICLEDATA_EXTERNTEMP" until="X.x">
 	<history>
-		<element name="VEHICLEDATA_EXTERNTEMP" />
+		<element name="VEHICLEDATA_EXTERNTEMP" since="1.0" until="X.x" />
```

Comment:
I think this should be since 2.0, per https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml#L536

Reply:
Done